docs/glossary.rst

```diff
@@ -0,0 +1,371 @@
+.. _glossary:
+
+----------------------
+Configuration Glossary
+----------------------
+
+Listed below are all configuration values that may be set for the QC dashboard.
+They're organized into sections based on the feature that they configure.
+These values should be set in your shell (or your uWSGI configuration file).
```

Comment:
I left out an example path here, because it's only relevant if the user does a full manual install (i.e. ignores the containers entirely) and differs by OS so at that point the user should probably know where it is anyway or they're going to run into a lot of other troubles too.